README.md

```diff
@@ -76,6 +76,14 @@ Add reviewers to the pull requests created by the GitHub Action. Multiple review
 
 By default, the GitHub Action scans the entire repository by looking for `project.clj` and `deps.edn` files. It is possible to define a specific sub-path to not scan the entire repository.
 
+### subdirectory_recursion
```

Comment:
I would call this something different, especially since it's either on/off and doesn't allow you to control the recursion depth.

Something like `search_subdirectories`

Reply:
I like `include_subdirectories`